packages/filter/src/actions/const.ts

```diff
@@ -0,0 +1,8 @@
+export const FILTERS_REMOVE_FILTER = '@xcritical-filter/REMOVE_FILTER';
```

Comment:
Question for discussion. If we have namespace, i am not sure it is necessary duplicate it name after /.
I mean `export const FILTERS_REMOVE_FILTER = '@xcritical-filter/REMOVE';` would be enough.

Reply:
Yes, you are rights